<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="tb_main"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>230</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Аэропорт:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cb_airports"/>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_arrival">
            <property name="text">
             <string>Прилёт</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_departure">
            <property name="text">
             <string>Вылет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Дата:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="de_date">
            <property name="maximumDate">
             <date>
              <year>2017</year>
              <month>9</month>
              <day>14</day>
             </date>
            </property>
            <property name="minimumDate">
             <date>
              <year>2016</year>
              <month>8</month>
              <day>15</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_getList">
            <property name="text">
             <string>Получить расписание</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_showLoad">
            <property name="text">
             <string>Отобразить загруженность</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_status">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_airports</tabstop>
  <tabstop>rb_arrival</tabstop>
  <tabstop>rb_departure</tabstop>
  <tabstop>de_date</tabstop>
  <tabstop>pb_getList</tabstop>
  <tabstop>pb_showLoad</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
